frontend/src/components/Review/Rating.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { ReactElement } from 'react';
+import styles from './Rating.module.scss';
+import FavoriteIcon from '@material-ui/icons/Favorite';
+import FavoriteBorderIcon from '@material-ui/icons/FavoriteBorder';
+
+type Props = {
+  readonly rating: number;
+};
+const filled_hearts = (n: number) => {
```

Comment:
`n` is not a very descriptive parameter name. It goes against what you typically do in interviews/algo questions but you should describe what the parameter actually represents so that the code is self-documenting.